src/main/java/com/gregtechceu/gtceu/data/tags/ItemTagLoader.java

```diff
@@ -34,6 +34,7 @@ public static void init(RegistrateTagsProvider<Item> provider) {
         create(provider, lens, Color.Blue, GTItems.MATERIAL_ITEMS.get(lens, Sapphire).getId());
 
         create(provider, CustomTags.TAG_PISTONS, Items.PISTON, Items.STICKY_PISTON);
+        create(provider, "personal_protective_equipment",Items.DIAMOND_HELMET.kjs$getIdLocation(),Items.DIAMOND_CHESTPLATE.kjs$getIdLocation(),Items.DIAMOND_LEGGINGS.kjs$getIdLocation(),Items.DIAMOND_BOOTS.kjs$getIdLocation()); //TODO equipment
```

Comment:
don't use the KubeJS methods.

Reply:
this wasn't meant to be in the commit anyway